mem-08-m.xml

```diff
@@ -8468,6 +9354,7 @@ Observe que o verbo usado para uma pulsação é {joq:v}. [AUTOTRANSLATED]</colu
       <column name="definition_ru">единица площади (27 квадратных {'ujmey:n @@'uj:n, -mey:n}) [AUTOTRANSLATED]</column>
       <column name="definition_zh_HK">面積單位（27平方{'ujmey:n@@'uj:n, -mey:n}） [AUTOTRANSLATED]</column>
       <column name="definition_pt">unidade de área (27 quadrados {'ujmey:n@@'uj:n, -mey:n})</column>
+      <column name="definition_fi">eräs pinta-alan yksikkö (27 neliö-{'uj:n}:ia})</column>
```

Comment:
Mismatched bracket. Also, is the ":ia" a plural indicator? I'm not sure how the embedding of foreign words work in Finnish, but is it not possible to just the entire word including the plural indicator in the original language (Klingon)?

Reply:
Generally in Finnish plural forms are not loaned from another languages. There are two different plural indicators in Finnish, the plurar proper and partitive. It is not enough to take the Klingon word. I know it looks a bit weird especially with neliö- (square) prefix, but I don't know any better way.

(There was some controversy when the translator of the Lord of the Rings took plural forms like "valar" and used them as such. My linguistics teacher was very disapproving...)